### ISSUE TITLE
Import-Module JiraPS - This script contains malicious content and has been blocked by your antivirus software

### ISSUE THREAD

**Issue by stan-spotts** [AtlassianPS/JiraPS#510]
## Description

Install-Module works fine, but when using Import-Module, receive an error saying the script has malicious content and was blocked by AV software.  Wondering what in the code is getting flagged as "malicious" and if there is a workaround. (Did ask IT to whitelist JiraPS in Carbon Black so that's pending)

## Steps To Reproduce

`Import-Module JiraPS`

## Expected behavior

Imported and ready to use module.

## Behavior
```
Import-Module JiraPS
At C:\Program Files\WindowsPowerShell\Modules\JiraPS\2.14.7\JiraPS.psm1:1 char:1
+ #region Dependencies
+ ~~~~~~~~~~~~~~~~~~~~
This script contains malicious content and has been blocked by your antivirus software.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : ScriptContainedMaliciousContent

Import-Module : The module to process 'JiraPS.psm1', listed in field 'ModuleToProcess/RootModule' of module
manifest 'C:\Program Files\WindowsPowerShell\Modules\JiraPS\2.14.7\JiraPS.psd1' was not processed because no valid
module was found in any module directory.
At line:1 char:1
+ Import-Module JiraPS
+ ~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ResourceUnavailable: (JiraPS:String) [Import-Module], PSInvalidOperationException
    + FullyQualifiedErrorId : Modules_ModuleFileNotFound,Microsoft.PowerShell.Commands.ImportModuleCommand
```

## Your Environment

```
Name   Version
----   -------
JiraPS 2.14.7

Name                           Value
----                           -----
PSVersion                      5.1.19041.4291
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.19041.4291
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1

```

**Comment by lipkau** [comment]
Closing as a duplicate of #527, which tracks signing the module assets so AV / endpoint-protection tools (Carbon Black, Defender, etc.) stop flagging them. Subscribe there for progress.
